extensions/amp-form/0.1/amp-form.js

```diff
@@ -376,8 +379,10 @@ export class AmpForm {
             /** @type {../../../src/service/xhr-impl.FetchResponse} */ (
                 error));
         rethrowAsync('Form submission failed:', error);
+        return error;
       });
     });
+    this.xhrSubmitPromise_ = p;
```

Comment:
Let's only set this in test mode, Closure can eliminate the property in prod, then.